client/src/styles/index.module.css

```diff
@@ -6,7 +11,7 @@
     background-size: cover;
     justify-content: center;
     align-items: center;
-    width: 100%;
+    width: 100vw;
```

Comment:
`100%` is fine, just move it to `.pageContainer`

Reply:
👍